src/sagas/wallet.js

```diff
@@ -381,8 +390,7 @@ export function* fetchTokensMetadata(tokens) {
   const tokenMetadatas = {};
   for (const response of responses) {
     if (response.type === types.TOKEN_FETCH_METADATA_FAILED) {
-      // eslint-disable-next-line
-      console.log('Error downloading metadata of token', response.tokenId);
+      log(`Error downloading metadata of token ${response.tokenId}.`);
```

Comment:
This will crash, log is an object with `warn`, `error` and `debug` methods

I've seen this in other files, can you please review?

Reply:
Fixed.